DocumentsFromSnapshotMigration/src/main/java/com/rfs/RfsMigrateDocuments.java

```diff
@@ -140,12 +153,21 @@ public static void main(String[] args) throws Exception {
 
         validateArgs(arguments);
 
+
+        var rootWorkCoordinationContext = new RootWorkCoordinationContext(
+                RootOtelContext.initializeOpenTelemetryWithCollectorOrAsNoop(arguments.otelCollectorEndpoint, "docMigrationCoordination"),
+                new CompositeContextTracker(new ActiveContextTracker(), new ActiveContextTrackerByActivityType()));
+        var rootDocumentContext = new RootDocumentMigrationContext(
+                RootOtelContext.initializeOpenTelemetryWithCollectorOrAsNoop(arguments.otelCollectorEndpoint, "docMigration"),
+                new CompositeContextTracker(new ActiveContextTracker(), new ActiveContextTrackerByActivityType()),
+                rootWorkCoordinationContext);
```

Comment:
There are multiple `root` contexts at the same time?  Lets drop the word root unless it means something, `WorkCoordinationContext` and `DocumentMigrationContext`

Reply:
Root does have meaning - the root context is a constructor and the root of the scope-trees for a given component.  Every scope needs a root, directly or indirectly.  From within the work coordination code (which should be considered its own independent component), there needs to be a root, hence the root coordination context.  

However, the doc migration code has its own containing context, hence the reason to splice the root and scopes from work coordination into the reindexing contexts when appropriate.

Reply:
[Offline Recap] - So yes this is generally true.  There is some slightly awkward terminology in the system, but there isn't any fundamental disagreement with how this works.  I'm fine moving forward with out any changes, resolving this thread.

Reply:
This is mostly true.  This part isn't quite...
> CoordinationContext is a child of the DocumentMigrationContext where it pulls configuration.

Both of those root contexts are 'root' to their callers.  Document migration contexts will be created from the root object or children that were returned by that root.  Wook coordination contexts will be created by the work coordination root context.   Those coordination contexts may have ancestor spans that are document migration spans.

For both 'root' contexts, they derive their configuration from the otel sdk, which, now in the latest code, is the same object - meaning that there's just one "service" in the otel-space for this process, which probably makes a lot more sense now.